src/coreclr/nativeaot/BuildIntegration/DotNetJsApi.targets

```diff
@@ -71,7 +71,7 @@
       <JSFileType Include="lib.js"     Kind="js-library" />
       <JSFileType Include="post.js"    Kind="post-js" />
       <JSFileType Include="extpost.js" Kind="extern-post-js" />
-      <_WasmExtraJSFile Include="$(IlcFrameworkNativePath)src\es6\*.%(JSFileType.Identity)" Kind="%(JSFileType.Kind)" />
+      <_WasmExtraJSFile Include="$(IlcSdkPath)src\es6\*.%(JSFileType.Identity)" Kind="%(JSFileType.Kind)" />
```

Comment:
I suspect this needs tweaks to `Microsoft.DotNet.ILCompiler.LLVM.pkgproj` to preserve the `src/es6`. Alternatively, it needs fixes here to work with the flag hierarchy (which would be simpler if it's possible).

Reply:
Good catch! I forgot changes to pkgproj in stash